Sources/Logging/Logging.swift

```diff
@@ -34,35 +34,35 @@ public struct Logger {
     }
 
     @inlinable
-    func log(level: LogLevel, message: @autoclosure () -> String, file: String, function: String, line: UInt) {
+    func log(level: LogLevel, message: @autoclosure () -> String, file: String = #file, function: String = #function, line: UInt = #line) {
         if self.logLevel <= level {
-            self.handler.log(level: level, message: message(), file: file, function: function, line: line)
+            self.handler.log(level: level, message: message(), error: nil, file: file, function: function, line: line)
         }
     }
 
     @inlinable
     public func trace(_ message: @autoclosure () -> String, file: String = #file, function: String = #function, line: UInt = #line) {
-        self.log(level: .trace, message: message(), file: file, function: function, line: line)
+        self.log(level: .trace, message: message, file: file, function: function, line: line)
```

Comment:
@weissi this seems to not have any functional difference but it "reads" better. wdyt?

Reply:
@tomerd I'd prefer not to take this because it's important to know where the `@autoclosure` is "resolved", ie. where the lazy value is forced. The `()` communicates this so I'd like to keep that if you don't mind.

Reply:
@weissi but then it should *not* be here, right? because we only want to resolve it after the log level check in L39 in this case? or am i missing something?